src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/ManagedTypeInfo.cs

```diff
@@ -51,6 +51,7 @@ public static ManagedTypeInfo CreateTypeInfoForTypeSymbol(ITypeSymbol type)
 
     public sealed record SpecialTypeInfo(string FullTypeName, string DiagnosticFormattedName, SpecialType SpecialType) : ManagedTypeInfo(FullTypeName, DiagnosticFormattedName)
     {
+        public static readonly SpecialTypeInfo Byte = new("byte", "byte", SpecialType.System_Void);
```

Comment:
@elinor-fung Are you sure `System_Void` is correct?

Reply:
Uh, definitely not. Thanks for catching this (luckily used as more of a placeholder right now, so not breaking anything).

Reply:
https://github.com/dotnet/runtime/pull/67261